.github/workflows/main.yml

```
@@ -136,8 +132,8 @@ jobs:
     needs:
       - test
       - docs
-      - check-links
-      - test_miniumum_verisons
+      - check_links
+      - test_minimum_verisons
```

ditto.  (Consistency is an issue in this case. :smile:)
```suggestion
      - test_minimum_versions
```